README.md

```diff
@@ -83,7 +83,7 @@ pathToEC2GOFile=src/main/resources/ec2go
 - The db.* values are required to make a database connection.
 - `testMode` can be set to `true` if you want to see what the results might look like, but not commit them to the database. Set this to false when you _do_ want the rsults to get committed.
 - person.id - This will be used as the Person ID for the InstanceEdits that this program creates.
-- pathToGOFile - the path to `gene_ontology_ext.obo`. This file contains the GO terms. You can download this file from [http://geneontology.org/ontology/obo_format_1_2/gene_ontology_ext.obo](http://geneontology.org/ontology/obo_format_1_2/gene_ontology_ext.obo).
+- pathToGOFile - the path to `gene_ontology_ext.obo`. This file contains the GO terms. You can download this file from [http://current.geneontology.org/ontology/go.obo](http://current.geneontology.org/ontology/go.obo).
 - pathToEC2GOFile - the path to `ec2go`. This file contains a mapping of EC numbers mapped to GO accessions. You can download this file from [http://geneontology.org/external2go/ec2go](http://geneontology.org/external2go/ec2go).
```

Comment:
For the links, if the link destination and the link text to display, markdown using square brackets and parentheses aren't necessary.  You can just put the literal link text in as ingle time and it will get rendered as a link.